recipes/dukpy/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "dukpy" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/amol-/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 081eda6d60053dc6b141171ff6cfd5dfd037ab06d4ad5fbeaf01b850029cac3d
+
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

you need to add the ``entry_points`` here
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  entry_points:
    - dukpy-install=dukpy.install:main
    - dukpy=dukpy.run:main
```